src/systems/filecoin_token/payment_channels/_index.md

```diff
@@ -10,7 +10,7 @@ entries:
 
 Payment Channels are used in the Filecoin {{<sref retrieval_market>}} to enable efficient off-chain payments and accounting between parties for what is expected to be series of microtransactions, specifically those occurring as part of retrieval market data retrieval.
 
-Note that the following provides a high-level overview of payment channels and an accompanying interface. The lotus implementation of [vouchers](https://github.com/filecoin-project/lotus/blob/master/chain/types/voucher.go) and [payment channels](https://github.com/filecoin-project/lotus/tree/master/paych) are also good references.
+Note that the following provides a high-level overview of payment channels and an accompanying interface. The lotus implementation of [vouchers](https://github.com/filecoin-project/lotus/blob/master/chain/types/voucher.go) and [payment channels](https://github.com/filecoin-project/lotus/tree/master/paychmgr) are also good references.
```

Comment:
Perhaps the link should go here instead: https://github.com/filecoin-project/specs-actors/tree/master/actors/builtin/paych